src/converter_test.rs

```diff
@@ -298,6 +298,23 @@ fn convert_line_cp_recursive() {
     assert_eq!(output, "xcopy /E directory\\sub1 director\\sub2");
 }
 
+#[test]
+#[ignore] // Multiple arguments are currently broken.
```

Comment:
lets remove these 2 tests until we can support it.
you can open a separate issue for this scenario